api-test/hasura/actions/_handlers/insertOrgMember.test.ts

```diff
@@ -0,0 +1,98 @@
+import { adminClient } from '../../../../api-lib/gql/adminClient';
```

Comment:
please move this to `api-lib/event_triggers/insertOrgMember.test.ts`

Reply:
`api-test` is only for code in `api`, because Vercel tries to make everything the `api` directory into an API route (an alternative would have been to put test files in there but add `_` to the beginning of their names)